a-star.ts

```diff
@@ -221,4 +223,11 @@ namespace scene {
         const c = locationCol(l);
         return tm.isObstacle(c, r);
     }
+
+    function isWalkable(l: tiles.Location, onTilesOf: Image, tm: tiles.TileMap): boolean {
+        if (isWall(l, tm)) return false;
+        if (!onTilesOf) return true;
+        const img = tm.getTileImage(tm.getTileIndex(l.col, l.row))
+        return img == onTilesOf;

```

Comment:
```suggestion
        return img.equals(onTilesOf);
```

in case they pass in their own redrawn image instead of the const?

Reply:
Agreed & fixed.